test/smoke/testApps/SpringBootTest/src/smokeTest/resources/appServers.txt

```diff
@@ -1 +1 @@
-tomcat8
\ No newline at end of file
+tomcat85
```

Comment:
@littleaj mostly springboot apps are deployed on Tomcat 8.5. It makes sense to test on it rather than tomcat 8.

Reply:
I added more web servers.

Reply:
I think it'll work on all of the current app servers. if they fail after the merge we can trim the list. Let's add back the other tomcat versions, jetty, jbosseap6 and jbosseap7 for now.

Reply:
@littleaj Done. Will reduce if things fail. By the way earlier it had only tomcat8 :) so we are expanding coverage.